static/search.js

```diff
@@ -1,3 +1,9 @@
+document.addEventListener('keyup', function (keyboardEvent) {
+    if (keyboardEvent.key === 's') {
```

Comment:
Maybe also capital `S`? 🤔

Reply:
In https://phpstan.org/ is even displaying the shortcut. I like that, maybe we could give this hint to the user, so they know already that it exists.
<img width="1004" alt="Screenshot 2023-02-02 at 17 52 01" src="https://user-images.githubusercontent.com/5256287/216389266-66632e05-3326-4a88-859a-4d239531a62c.png">

Reply:
I added S and /, so everyone is happy now 🤣 
@JesusValera and I will investigate this further to have something to help the user know that this exists (right now it could be an "easter egg" 🍳